parser.go

```diff
@@ -804,7 +804,7 @@ func (p *parser) parseParamComment(pkgPath, pkgName string, operation *Operation
 	// {name}  {in}  {goType}  {required}  {description}
 	// user    body  User      true        "Info of a user."
 	// f       file  ignored   true        "Upload a file."
```

Comment:
Do any of the example comments need to be updated?

Reply:
I don't think this would be a good place to document how to use `anyOf/allOf/oneOf/not` - I could add something to the readme maybe?

Reply:
yeah I wasn't thinking full documentation there just an example. But readme is probably a better spot, even if we just call out that it is supported.

Reply:
Added